components/esp32_camera.rst

```diff
@@ -148,6 +148,30 @@ Configuration for M5Stack Camera
       # ...
 
 
+Configuration for M5CameraF Camera (c.f. https://github.com/m5stack/m5stack-cam-psram)
```

Comment:
No links in titles please - it will not show up properly.

Please remove the link from the title and move it below in a short sentence before the code block.